frontend/src/components/browse-sightings-page/SightingCard.tsx

```diff
@@ -11,8 +11,19 @@ export const SightingCard: React.FunctionComponent<SightingProps> = ({
 }) => {
   return (
     <div className="sighting-card">
-      <h3 className="fade-in">Sighting #{sighting.id}</h3>
+      {!sighting.species?.name ? (
+        <h3 className="sighting-card_title fade-in">
+          Sighting recorded by {sighting.seenBy}
+        </h3>
+      ) : (
+        <h3 className="sighting-card_title fade-in">
+          Sighting of {sighting.species?.name}
+        </h3>
+      )}
```

Comment:
Avoid the "`!`" and swap these two:
```suggestion
      {sighting.species?.name ? (
        <h3 className="sighting-card_title fade-in">
          Sighting of {sighting.species?.name}
        </h3>
      ) : (
        <h3 className="sighting-card_title fade-in">
          Sighting recorded by {sighting.seenBy}
        </h3>
      )}
```

Reply:
makes sense, thanks!